test/system/src/test/java/io/pravega/test/system/MultiReaderTxnWriterWithFailoverTest.java

```diff
@@ -329,11 +332,6 @@ private void performFailoverTest() throws InterruptedException {
 
         log.info("Read count: {}, write count: {} without any failover after sleep before scaling", currentReadCount2, currentWriteCount2);
 
-        //ensure writes are happening
```

Comment:
If we compare the post failover write count and pre failover write count , without waiting for transaction status, it will fail if transactions are partially committed.  So, we are doing a ` FutureHelpers.allOf(txnStatusFutureList).get();`  at the end of test , before comparing read and write count.